docs/knowledgebase/getting-started/glossary.md

```diff
@@ -157,6 +163,13 @@ There is
 [a comprehensive blog series](https://polkadot.network/polkadot-consensus-part-1-introduction/) that
 provides a deep dive into the consensus strategies of [the Polkadot Network](#polkadot-network).
 
+### Hybrid Consensus
+
+A blockchain consensus protocol that consists of independent or loosely coupled mechanisms for
+[block production](#author) and [finality](#finality). This allows the chain to grow as fast as
+probabilistic consensus protocols, such [Aurand](#aurand), while still maintaining the same level of
+security as [instant finality](#instant-finality) consensus protocols.
```

Comment:
Also block production algorithms tend to be O(n), while finality gadgets tend to be O(n^2). Separating them allows the chain to progress with lower communication overhead while choosing an acceptable `L`, or lag to finality, to make finalization O(n^2/L) by finalizing in batches separated by `L`. Not sure if this is within the scope of the glossary or should be covered elsewhere.